mockk/jvm/src/test/kotlin/io/mockk/it/MockStaticMethodTest.kt

```diff
@@ -2,13 +2,43 @@ package io.mockk.it
 
 import io.mockk.every
 import io.mockk.mockkStatic
+import io.mockk.unmockkStatic
 import io.mockk.verify
+import org.junit.Test
+import java.time.Instant
 import kotlin.test.Test
```

Comment:
You are importing org.junit.Test, this should be removed

Reply:
Thanks. It was imported for Issue92Test, i didn't realize that was a kotlin class for same reason.